Lib/test/test_sys.py

```diff
@@ -695,10 +692,8 @@ def test_43581(self):
         self.assertEqual(sys.__stdout__.encoding, sys.__stderr__.encoding)
 
     def test_intern(self):
-        global INTERN_NUMRUNS
-        INTERN_NUMRUNS += 1
         self.assertRaises(TypeError, sys.intern)
-        s = "never interned before" + str(INTERN_NUMRUNS)
+        s = "never interned before" + str(random.random())
```

Comment:
I would prefer using random.randint(), for example 10**9 to have 9 random digits. I prefer to avoid float when possible 😁 Same below.

Reply:
Done

Reply:
On my machine, the `random.random()` returns somethin like `0.35934878814573135`, which have 18 digits. So can we just increase the max range to `10 ** 18` to keep the random resolution almost like as the `random.random()`?